core/ast.h

```diff
@@ -646,7 +647,7 @@ struct ObjectField {
 
     ObjectField(enum Kind kind, const Fodder &fodder1, const Fodder &fodder2,
                 const Fodder &fodder_l, const Fodder &fodder_r, enum Hide hide, bool super_sugar,
-                bool method_sugar, AST *expr1, const Identifier *id, const ArgParams &params,
+                bool method_sugar, AST *expr1, const Identifier *id, const LocationRange &id_lr, const ArgParams &params,
```

Comment:
nit: add a newline after `id_lr,`, put params on the next line together with `bool trailing_comma`.

Reply:
ah, yes